src/validations/rules/rules.xsl

```diff
@@ -125,8 +125,7 @@
                 <!-- that's correct --></xsl:when>
             <xsl:otherwise>
                 <xsl:message
-                    expand-text="true"
-                    terminate="true">{base-uri()} is not static-base-uri()</xsl:message>
```

Comment:
We removed this `terminate` with regret, but it fixes a minor mismatch of paths that exists when properly running this transform on Windows.